### PR TITLE
Filter,Logging,DownloadFile.

### PR DESCRIPTION
**Pull request by MaximIsaev**
Added:
1.Filter by author name;
2.Logging through Listeners(for example added 3 Listeners);
3.Download news feed xml file from server, so you mau not have complete file on your pc, it will download automatically to "..\webapps\NewsData\NewsStorage\NewsFeed.txt";
About static variables:
Because many classes won't create their instance, methods and fields execute from class and if I even removed all modifier "static" the IDE will swear(ругаться) to return it back;
About HTML tags:
Some tags(list tags)  must be created dynamically , because their count depends from quantity of folder content;.

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -26,20 +26,4 @@
   <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" assert-keyword="true" jdk-15="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/out" />
   </component>
-  <component name="masterDetails">
```
Comment (ferustigris) [inline .idea/misc.xml:29]: Are this files really should be in git?

File: pom.xml [hunk 2]
```diff
@@ -8,22 +8,71 @@
...
         </dependency>
         <dependency>
-            <groupId>org.json</groupId>
-            <artifactId>json</artifactId>
-            <version>20150729</version>
+            <groupId>com.googlecode.json-simple</groupId>
+            <artifactId>json-simple</artifactId>
+            <version>1.1</version>
+        </dependency>
+        <!--<dependency>-->
```
Comment (ferustigris) [inline pom.xml:59]: unused code should be removed

File: src/main/java/com/get_news_feed_file/DownloadNewsFeedFile.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    public static File getOutputFeedFile() {
+        return outputFeedFile;
+    }
+
+    public static boolean validateOutputFeedFile() {
+
+        try {
+            if (createNewsFolder()) {
+                if (!outputFeedFile.exists()) {
+                    outputFeedFile.createNewFile();
```
Comment (ferustigris) [inline src/main/java/com/get_news_feed_file/DownloadNewsFeedFile.java:47]: too many "if", try to extract some logic in different method

File: src/main/java/com/json_news_item/JSONContainer.java [hunk 4]
```diff
@@ -1,27 +1,41 @@
...
-        for (int i = 0; i < jsonNewsList.size(); i++) {
-            out.println(jsonNewsList.get(i));
-        }
-        out.close();
 
+
+        out.print(jsonNewsList.toJSONString());
+    }
+
+    public static String encodeHtmlTag(String tag) {
```
Comment (ferustigris) [inline src/main/java/com/json_news_item/JSONContainer.java:26]: do we have standard class for this work?
http://docs.oracle.com/javase/7/docs/api/java/net/URLEncoder.html?

Comment (MaximIsaev) [inline src/main/java/com/json_news_item/JSONContainer.java:26]: I tried to use it,but this class for encoding URL, but I just need to encode HTML tags(for just simple displaying as text symbol). And this class encode all of it in symbol format "%xy", so this class doesn't do what I need.

File: .idea/misc.xml (hunk 1 above)
Comment (MaximIsaev) [inline .idea/misc.xml:29]: I'm sorry. My mistake, their shouldn't be in GIT.

**Pull request closed without merging**